README_zh_CN.md

```diff
@@ -76,9 +76,9 @@ mc.exe --help
 ```
 
 ## 通过源码安装
-通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从https://minio.io/downloads/#minio-client下载官方版本。
+通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从https://min.io/downloads/#minio-client下载官方版本。
 
```

Comment:
The url is unreachable. `https://min.io/download/#minio-client` is the correct one.

Reply:
Done.